### PR TITLE
Hyperlink fixes

### PR DESCRIPTION
**Pull request by malw4**

### CODE REVIEW

**Comment by malw4** [comment]
@JZimnol can you check is there are any hyperlinks to fix?

**Comment by JZimnol** [comment]
@malw4 a few notes:
- there is a following warning:
  ```
  WARNING  -  Documentation file 'Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Device_operations/Upgrading_firmware.md' contains a link to 'Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Device_operations/images/device_inventory_view.png' which is not found in the documentation files.
  ```
  the file `device_inventory_view.png` does not exist in the repository. Maybe someone didn't add it or someone deleted it, anyway it does not exist and is displayed like this:
  image
  Maybe you should contact someone who is responsible for `Azure_IoT` pages to fix this?
- you've changes some of the URLs from e.g `LwM2M_Client/ESP32/M5Stick` to `https://iotdevzone.avsystem.com/docs/LwM2M_Client/ESP32/M5Stick` which is cool, but you did that only in some places. In some places you've left the relative paths (with `./` or `../`). I'd prefer some consistency with that (I think the absolute paths are OK)
- the Svetovid's tutorial is missing in the rendered docs (the file exist in the repo `Anjay_integrations/RaspberryPi_integration/Implementing_LwM2M_objects_on_RaspberryPi.md` but is missing in the `mkdocs.yml` file). I'm not sure if that file (`mkdocs.yml`) file should be edited manually, so ping: @laurensslats

File: docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Device_operations/Upgrading_firmware.md [hunk 1]
```diff
@@ -148,4 +148,4 @@ To cancel the firmware upgrade procedure, follow these steps:
 
 ## See also
 
-See the relevant section of [LwM2M mappings](/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings_Hub/#lwm2m-executable-resources) to learn the details of how Azure IoT Hub Direct Methods are mapped in Coiote DM.  
+See the relevant section of [LwM2M mappings](https://iotdevzone.avsystem.com/docs/Cloud_integrations/AWS_IoT_Core/Concepts/AWS_Integration_concepts/) to learn the details of how Azure IoT Hub Direct Methods are mapped in Coiote DM.
```
[the remark this answers is not in the document]
Comment (laurensslats) [reply docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Device_operations/Upgrading_firmware.md]: already updated

File: docs/LwM2M_Client/Nordic/Thingy91.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ To program the board, go through the process of **flashing Thingy:91**. Use the
 
 After successful flashing, reboot the board and go to the next step.
 
-## Connect to the LwM2M Server
+## Connecting to the LwM2M Server
```
Comment (laurensslats) [inline docs/LwM2M_Client/Nordic/Thingy91.md]: Keep in present tense

**Review by laurensslats — APPROVED** [review]
Approved with 2 small edits which I already updated

**Comment by JZimnol** [comment]
The `Svetovid` tutorial is still not visible in the rendered docs:
```
INFO     -  The following pages exist in the docs directory, but are not included in the "nav"
            configuration:
              - cloud-integrations.md
              -
            Anjay_integrations/RaspberryPi_integration/Implementing_LwM2M_objects_on_RaspberryPi.md
```

**Comment by laurensslats** [comment]
@JZimnol 

The `Svetovid` is visible, but it has moved to a different url. See: https://iotdevzone.avsystem.com/docs/Demo_Projects/Implementing_LwM2M_objects_on_RaspberryPi/

For some reason, there are 2 different files in the DevZone which are not 100% the same. Let me remove the oldest version.

**Pull request merged**